pkg/global/apply_configuration.go

```diff
@@ -44,9 +45,14 @@ func ApplyConfiguration(configuration *pb.Configuration) error {
 		}
 
 		if stackdriverConfiguration := tracingConfiguration.Stackdriver; stackdriverConfiguration != nil {
+			hostname, err := os.Hostname()
+			if err != nil {
+				return err
+			}
 			se, err := stackdriver.NewExporter(stackdriver.Options{
-				ProjectID: stackdriverConfiguration.ProjectId,
-				Location:  stackdriverConfiguration.Location,
+				ProjectID:              stackdriverConfiguration.ProjectId,
+				Location:               stackdriverConfiguration.Location,
+				DefaultTraceAttributes: map[string]interface{}{"hostname": hostname},
```

Comment:
Main concern is how to handle places where hostname is valuable, i.e. I've got a statefulset of storage nodes in kubernetes, named storage-0 through storage-2, tracking stuff to a specific node is really helpful.  I suppose this could be something where we lean on jsonnet to template this on a per node basis?

Reply:
Yep! Just use [this](https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/) to assign environment variables. You can then use `std.extVar('FOO')` in Jsonnet to access those. That's also how we construct worker IDs right now.